sample-plugin/pom.xml

```diff
@@ -33,6 +33,10 @@
       <groupId>io.jenkins.plugins</groupId>
       <artifactId>byte-buddy-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>eddsa-api</artifactId>
```

Comment:
Note: the original intent of this file was to include only dependencies that were actually used “naturally”, not transitive deps. This was never mechanically enforced, though, and has proven difficult to manage: #683